src/index.js

```diff
@@ -28,6 +29,7 @@ addFilter( 'native.block_editor_props', 'gutenberg-mobile', ( editorProps ) => {
 		return {
 			...editorProps,
 			initialTitle,
+			initialData: initialHtml,
```

Comment:
Maybe it's better to append `initialHtml` from gutenberg to this like: `initialHtmlGutenberg + initialHtml` so we have both when running from gb-mobile.

Reply:
sounds like a good idea! would like to be able to see all blocks together

Reply:
I was thinking of how we could achieve this:
1. Add another filter (`addFilter`) that executes after [gutenberg's filter](https://github.com/WordPress/gutenberg/blob/51233ab706a9630e997cc9a508f72a9fe99a27ca/packages/react-native-editor/src/index.js#L63) — override the title and append the Jetpack blocks to the content
2. Export Gutenberg's `initial-html` and import it here into gutenberg-mobile

For (1), the challenge is how to allow the main apps to set the title and content without them being overriden. For (2), in my tests merely importing the `react-native-editor` instantiates it, so I couldn't simply export `initial-html` from inside gutenberg and import it in gutenberg-mobile

Do y'all have any thoughts on how this can be done? Maybe I'm overlooking something.

Reply:
> I couldn't simply export initial-html from inside gutenberg and import it in gutenberg-mobile

This seems the most natural thing to do to me, but I have trouble understanding why (2) is not possible, I think I don't fully understand how exporting that in GB is not import-able in mobile-gb?

Reply:
Looks like (2) works, I just needed to make a few tweaks. PR incoming.